attestation-evidence-broker/docs/design/design.md

```diff
@@ -0,0 +1,121 @@
+
+# Background of SEV(-ES) runtime remote attestation
+
+Compared with SEV SNP and Intel TDX, the runtime remote attestation of SEV(-ES) does NOT support guest-provided data to be included into the attestation report [1,2]. In addition, SEV(-ES) does NOT support dynamic measurement and can only verify static measurement values, which are always constant values. Although SEV(-ES) has this limitation, runtime remote attestation is still useful. When the SEV(-ES) guest is running, it can report the attestation evidence with fixed launch digest as a heartbeat for trusted healthy check.
+
+SEV(-ES) remote attestation includes two parties: attester and verifier.
+
+- The attester running in the SEV(-ES) guest is responsible for collecting attestation evidence.
+- The verifier runs in the trusted domain of the tenant and is responsible for verifying the attestation evidence sent by the attester. Noted that **verifier can run on any platform (including non-SEV(-ES) platforms)**.
+
+What SEV(-ES) attestation evidence should be collected by the attester?
+
+# The content of SEV(-ES) Attestation Evidence
+
+Verifier uses the following SEV(-ES) certificate chain to verify the integrity of the attestation report generated by the `ATTESTATION` command [2]:
+```bash
+ARK -> ASK -> CEK -> PEK -> report
+OCA -> PEK -> report
+```
+
+- `foo -> bar` means to use foo's public key to verify the integrity of bar.
+- ARK is the root of trust for AMD, OCA is the root of trust for platform owners. OCA has two ways:
+    1. Self-owned: The OCA Key Pair and self-signed OCA certificate are automatically generated by the SEV(-ES) firmware.
+    2. Externally-owned: External users use OCA Key Pair to generate self-signed OCA certificates in a trusted environment.
+
+Verifier needs to verify the attestation report with the certificate chain on any platform. ARK and ASK can be obtained directly from the AMD KDS server. CEK, PEK, OCA, and attestation report are related to the specific SEV(-ES) platform, therefore **SEV(-ES) Attestation Evidence collected by attester should include: attestation report, PEK, CEK, OCA certificate. **
+
+| Contents of SEV(-ES) Attestation Evidence | SEV(-ES) firmware instruction |
+| ------------------------------------------| ------------------------ |
+| attestation report                        | ATTESTATION              |
+| CEK                                       | GET_ID                   |
+| OCA,PEK                                   | PDH_CERT_EXPORT          |
+
+## The methods of query SEV(-ES) Attestation Evidence
+
+According to the [officially infomation](https://github.com/AMDESE/AMDSEV/issues/71#issuecomment-926118314) provided by AMD, **SEV(-ES) firmware APIs don't support query attestation report in SEV(-ES) guest and there is no plan to support it in the future. New features will be added to the SNP APIs instead.**
```

Comment:
s/officially infomation/official feedback/
s/provided by AMD//
s/don't/doesn't/
s/New features will be added to the SNP APIs instead./Instead, this capability will be available in SEV-SNP./

Reply:
The word of `APIs` is plural， the statement about `APIs don't ` looks Reasonable。

The statement about `there is no plan to support it in the future. New features will be added to the SNP APIs instead` is refer to AMD  Developer
![image](https://user-images.githubusercontent.com/26923638/148217986-caa9ff1e-16a9-4755-bcc3-61241e271a69.png)

Reply:
I mean it is not necessary to quote the original statements in any case, because you are mentioning it in your context. The so-called "new feature" is not appropriate in your context.